ztp/ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator/policyGen/policyBuilder.go

```diff
@@ -37,39 +41,50 @@ func (pbuilder *PolicyBuilder) Build(policyGenTemp utils.PolicyGenTemplate) (map
 			if err != nil {
 				return policies, err
 			}
-			name := policyGenTemp.Metadata.Name + "-" + sFile.PolicyName
-			path := policyGenTemp.Metadata.Name + "/" + policyGenTemp.Metadata.Name + "-" + sFile.PolicyName
 			if sFile.PolicyName == "" {
```

Comment:
Generating non-policy wrapped CRs is something that is extremely helpful for debug and very useful when working with single clusters. Would it make sense to add a global flag (command line switch) which we || here?

Reply:
I'd love that, but that's a bit out of scope for what I want this hack to be.  That can be considered more fully in https://issues.redhat.com/browse/CNF-2885